pages/wallets/[wallet].tsx

```diff
@@ -111,6 +111,7 @@ function Wallet(props: { address: string }) {
 
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [router]);
+
```

Comment:
Maybe remove this page. This page that us have a quick link to each wallet
- `/wallets/ <multisig wallet address>`

In new design, we should also keep having routes. for example
-  `/<multisig wallet address>/proposals`
- `/<multisig wallet address>/history`
-  etc ...

Reply:
Note, currently, after finish creating/importing a wallet, there is a button to go  `/wallets/ <multisig wallet address>`.

Reply:
Hmm it's a good idea and it's not that complicated, but it would require some refactor to achieve it, so let's keep it for another PR